docs/developerguide.rst

```diff
@@ -2,3 +2,35 @@
 
 Developer guide
 ===============
+
+This tutorial explains how to locally-start REANA-UI, with automatic refresh:
```

Comment:
Could we better use the word `guide` than `tutorial`? It is what we are using everywhere else, plus it is more appropriate. Also after googling a bit, I found live reloading to be the common term when it comes to talk about the automatic refresh, shall we switch 😜 ?